frontend/src/router/index.ts

```diff
@@ -1,20 +1,21 @@
 import Vue from 'vue';
 import VueRouter, { RouteConfig } from 'vue-router';
-import NewGame from '../views/NewGame.vue';
-import AccessGame from '../views/AccessGame.vue';
+
+import Home from '../views/Home.vue';
+import Game from '../views/Game.vue';
 
 Vue.use(VueRouter);
 
 const routes: RouteConfig[] = [
   {
     path: '/',
-    name: 'NewGame',
-    component: NewGame,
+    name: 'Home',
+    component: Home,
```

Comment:
`Home` feels a little odd, `Join`? `New`?

Reply:
But it's the root page, the starting point of the app. `Start` is the only other term I can think of.

Reply:
Now reflecting about it, I think that `Home` is the most descriptive term for this component. 

Also, from wikipedia's "[Home page](https://en.wikipedia.org/wiki/Home_page)": 

> The term can also refer to one or more pages always shown in a web browser when the application starts up. In this case, it is also known as the start page.

Which is exactly what this is about.

Reply:
Oooookay.